extensions/wikia/Wall/Wall.setup.php

```diff
@@ -224,15 +224,21 @@
 $wgGroupPermissions['vstf']['wallarchive'] = true;
 $wgGroupPermissions['helper']['wallarchive'] = true;
 $wgGroupPermissions['sysop']['wallarchive'] = true;
+$wgGroupPermissions['moderator']['wallarchive'] = true;
 
 $wgGroupPermissions['*']['wallremove'] = false;
-$wgGroupPermissions['user']['wallremove'] = true;
+$wgGroupPermissions['user']['wallremove'] = false;
```

Comment:
Hmm, right you are.

Looks like it's from [this line](http://pbrd.co/1HPvWfv) in CommonSettings, which is annoying because it means we're setting permissions for this extension in 2 different places. In fact, apart from that one line this [entire block](http://pbrd.co/1HPweD7) in CommonSettings is duplicated and then expanded on in Wall.setup.php [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Wall/Wall.setup.php#L213-L244).

That doesn't make any sense. Let's just delete that block from CommonSettings and then remove lines 230 and 265 from this file.

@sactage, are you able to see/branch off our config repo? (I wasn't sure, that's why I linked images instead of github links above). If so, can you create a branch and make that change? If not, I can go ahead and do it. Either way, please ahead and remove lines 230 and 265 from this file.